### PR TITLE
TestServerResolver: add CNAME test.

### PR DESCRIPTION
**Pull request by mkrautz**
In mumble-voip/mumble#3163 it was reported that a server that was
accessible via a CNAME was no longer being resolved correctly by
Mumble's connect dialog.

This commit adds CNAME to the ServerResolver test suite, so we're
sure that it is working.

### CODE REVIEW

**Comment by Kissaki** [comment]
```
FAIL!  : TestServerResolver::simpleAAAA() Compared values are not the same
   Actual   (spy.count()): 0
   Expected (1)          : 1
   Loc: [TestServerResolver.cpp(172)]
```
Here we go. #3162

File: src/tests/TestServerResolver/TestServerResolver.cpp [hunk 1]
```diff
@@ -68,6 +69,52 @@ void TestServerResolver::simpleSrv() {
...
+	ServerResolver r;
+	QSignalSpy spy(&r, SIGNAL(resolved()));
+
+	QString hostname = QString::fromLatin1("cname.serverresolver.mumble.info");
+	quint16 port = 64738;
+
+	r.resolve(hostname, port);
+
+	// Equivalent to QSignalSpy::wait() in Qt 5.
+	{
```
Comment (Kissaki) [inline src/tests/TestServerResolver/TestServerResolver.cpp]: Now that #3170 landed should call `signalSpyWait` (need to rebase onto new master).

**Comment by Kissaki** [comment]
Not sure that’s what #3163 is about (as he still has the issue after removing cname), but test still makes sense.

**Comment by mkrautz** [comment]
#3163 is fixed by https://github.com/mumble-voip/mumble/pull/3172

**Comment by mkrautz** [comment]
@Kissaki PTAL.

**Pull request merged**